README.md

```diff
@@ -14,19 +14,44 @@ bulky image viewer ... and don't care about resolution.
 
 ### Install
 
+First get the repository and change into the `src` directory
+
 ```bash
 git clone https://github.com/hzeller/timg.git
 cd timg/src
+```
+
+Compiling on Debian/Ubuntu
+```
```

Comment:
if you label the code block with the language in question (bash), then it will properly color encode the comment

```
  ```bash
```

Reply:
Done